pallet/src/lib.rs

```diff
@@ -32,16 +43,37 @@ pub mod pallet {
             + RuntimeAppPublic
             + Default
             + MaybeSerializeDeserialize;
+        type Event: From<Event<Self>> + IsType<<Self as frame_system::Config>::Event>;
+    }
+
+    #[pallet::event]
+    #[pallet::metadata(T::AccountId = "AccountId")]
+    #[pallet::generate_deposit(pub(super) fn deposit_event)]
+    pub enum Event<T: Config> {
+        ChangeValidators(Option<Vec<T::AccountId>>),
     }
 
+    pub struct AlephSessionManager<T>(sp_std::marker::PhantomData<T>);
+
     #[pallet::pallet]
     pub struct Pallet<T>(sp_std::marker::PhantomData<T>);
 
     #[pallet::hooks]
     impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {}
 
+    //ensure_root
```

Comment:
The first line in the below call should be: `frame_system::ensure_root(origin)?`.